app/javascript/packs/base.js

```diff
@@ -14,21 +14,20 @@ function getPageEntries() {
   });
 }
 
-const menus = [...document.getElementsByClassName('js-hamburger-trigger')];
-const moreMenus = [...document.getElementsByClassName('js-nav-more-trigger')];
+function initializeNav() {
```

Comment:
Thanks for making these changes @Zhao-Andy.

For testing this, we could add an end to end test, but I'm wondering if it's worth the effort for this particular fix. Maybe in a separate PR we could have an end to end test that tests the sidebar navigation. 🤔

Reply:
@nickytonline I spent about 30 minutes trying to do this and I think I need to seed some custom data. Not sure the best way to do that, but something like:

```js
cy.exec("rails db:seed")
```

would work. However, our particular seed file doesn't have the data I need `NavigationLink`. Is there a pattern we seed data with Cypress already? If not I think we could make that!

Reply:
There's an e2e seed file where you can put that stuff. Currently I je seed file is pretty bare bones as I've only added what needs to be there for e2e tests.

Reply:
Oh great, what's that file called?

Reply:
These [E2E docs](https://docs.forem.com/tests/e2e-tests/) should help as well.